shops/src/main/resources/shops-migrations/V15_5__Add_prismarine.sql

```diff
@@ -0,0 +1,7 @@
+
+INSERT IGNORE INTO shopitems
+(Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice)
+VALUES
+('Resources', 'PRISMARINE_SHARD', 'Prismarine Shard', 0, 10, 2, 1, 10, 1),
+('Resources', 'PRISMARINE_CRYSTALS', 'Prismarine Crystals', 0, 11, 2, 1, 10, 1)
```

Comment:
Dark prismarine is a extra protective block from cannons, perhaps make it uncraftable

Reply:
Yeah I think that's the best option, then sell it elsewhere